app/views/my/_action_menu.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="btn-group">
+  <button class="btn btn-sm dropdown-toggle" data-toggle="dropdown" type="button" id="dropdownMenu_<%= id %>" aria-haspopup="true">
+    <span class="sr-only">Press to </span>
+    Select an action
+    <span class="caret" aria-hidden="true"></span>
+  </button>
+  <ul role="menu" class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenu_<%= id %>">
+    <li role="menuitem" tabindex="-1">
+    <%= link_to raw('<i class="glyphicon glyphicon-link over" aria-hidden="true"></i> Single-Use Link to File'), '#',
+      class: "copypaste itemicon itemcode", title: "Single-Use Link to File", id: "copy_link_#{id}" %>
+    </li>
+    <li role="menuitem" tabindex="-1">
+    <%= link_to raw('<i class="glyphicon glyphicon-download-alt" aria-hidden="true"></i> Download File'), sufia.download_path(id),
+      class: 'itemicon itemdownload', title: 'Download File', target: '_new' %>
+    </li>
+    <li role="menuitem" tabindex="-1">
+      <%= display_trophy_link(@user, id) do |text| %>
+        <i aria-hidden="true" class='glyphicon glyphicon-star'></i> <%= text %>
+      <% end %>
+    </li>
+  </ul>
+</div>
```

Comment:
Is this a template to override another a default one where any mention of create/edit/delete has been removed?

Reply:
Yes, exactly that. To find the default templates I've been referencing this branch of the Sufia project: https://github.com/samvera-deprecated/sufia/tree/6.x-stable